engines/experiment/pkg/request/request.go

```diff
@@ -60,7 +60,7 @@ func GetValueFromHTTPRequest(
 	case PayloadFieldSource:
 		return getValueFromJSONPayload(bodyBytes, field)
 	case HeaderFieldSource:
-		value := reqHeader.Get(field)
+		value := strings.Join(reqHeader.Values(field), ",")
```

Comment:
This is required for array/object. This is backward compatible (as seen from unit test) if there is only 1 element